versions.sh

```diff
@@ -6,6 +6,7 @@ declare -A otpMajors=(
 	[3.9]='25'
 	[3.10]='25'
 	[3.11]='25'
+	[3.12]='25'
```

Comment:
It probably doesn't make sense to update this to the 26.x pre-releases, right? :joy:  (seems a bit pre-mature)

Any other changes we should consider for 3.12+ right now?

Reply:
It does not and there is a known incompatibility in the JSON generator we use